common/build/eslint-config-fluid/minimal.js

```diff
@@ -57,13 +57,20 @@ module.exports = {
 		"unused-imports",
 		// Plugin documentation: https://www.npmjs.com/package/eslint-plugin-unicorn
 		"unicorn",
+		// Custom ESLint rules
+		"custom-rules",
```

Comment:
If we rename our fake plugin "package" `eslint-plugin-fluid`, then I think this just becomes "fluid", and our rules get used as "fluid/foo".

Reply:
I personally think `fluid/foo` (e..g, `fluid/no-member-release-tags`) can be misleading. Maybe change the name to something like `eslint-plugin-fluid-custom-rules` so that the custom rules can appear as `fluid-custom-rules/no-member-release-tags` in config files?

Reply:
That works too. Just so long as it's obviously an internal fluid thing and not from some external library.